undertow/src/test/java/org/wildfly/extension/undertow/AbstractUndertowSubsystemTestCase.java

```diff
@@ -34,14 +34,10 @@
 
 public abstract class AbstractUndertowSubsystemTestCase extends AbstractSubsystemSchemaTest<UndertowSubsystemSchema> {
     final Map<ServiceName, Supplier<Object>> values = new ConcurrentHashMap<>();
-    private final UndertowSubsystemSchema schema;
-
-    AbstractUndertowSubsystemTestCase() {
-        this(UndertowSubsystemSchema.CURRENT);
-    }
+    protected final UndertowSubsystemSchema schema;
 
     AbstractUndertowSubsystemTestCase(UndertowSubsystemSchema schema) {
-        super(UndertowExtension.SUBSYSTEM_NAME, new UndertowExtension(), schema, UndertowSubsystemSchema.CURRENT);
+        super(UndertowExtension.SUBSYSTEM_NAME, new UndertowExtension(), schema, UndertowSubsystemSchema.CURRENT.get(schema.getStability()));
```

Comment:
Do you have any tests that ensure that schema versions 13 and earlier are also parsed correctly? 
If you would like to to that without adding a new class, you can use `@RunWith(Parameterized.class)` tag at the beginning of the class and the parameters can be `EnumSet.allOf(UndertowSubsystemSchema.class)`

Reply:
AFAIR comment in code suggests its not done, neither RFE is about that :)

Reply:
Sounds good!